src/Console/Command/ProcessCommand.php

```diff
@@ -185,7 +185,7 @@ private function processFile(SplFileInfo $fileInfo): void
     {
         $oldContent = $fileInfo->getContents();
 
-        if ($this->parameterProvider->provideParameter(Option::OPTION_DRY_RUN)) {
+        if ($this->parameterProvider->provideParameter(Option::OPTION_DRY_RUN) !== null) {
```

Comment:
I don't see point in turning everything `if ($value)` into `if ($value !== null)`
It the return value would be -5, false, null and some value, it would make sense. Now it only forces more writing and more thinking while reading the code: "what other false value apart `null` that method returns?"

Could you provide some reasoning?

Reply:
PHPStan complained about that nothing more than booleans should be passed to an `if` statement.

The `ParameterProvider::provideParameter` method returns the given parameter **or null**, so we need to check if the parameter is different from `null`, right?

Reply:
Well, this is in application flow could not be used outside, so PHPStan can't correctly detect this.